plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/providers/vfs/VFSFileProvider.java

```diff
@@ -88,6 +126,26 @@ public class VFSFileProvider extends BaseFileProvider<VFSFile> {
     return TYPE;
   }
 
+  /**
+   * Determines is a <code>filePath</code> is a VFS file
```

Comment:
small typo, first "is" instead of "if"

Reply:
addressed in https://github.com/pentaho/pentaho-kettle/commit/8d2d79b16445a6dfdaf944f34a702997b3bfe6b5